--- a/src/callproc.c.orig	2021-12-24 19:36:36.725590812 +0900
+++ b/src/callproc.c	2021-12-24 19:35:48.875295993 +0900
@@ -22,6 +22,7 @@
 #include <config.h>
 #include <errno.h>
 #include <stdlib.h>
+#include <sys/resource.h>
 #include <sys/types.h>
 #include <unistd.h>
 
@@ -1637,6 +1638,10 @@
       eassert (0 < vfork_error);
       return vfork_error;
     }
+  else
+    {
+      setpriority(PRIO_PROCESS, pid, 0);
+    }
 
   eassert (0 < pid);
   *newpid = pid;
